components/generic/SocialShare.vue

```diff
@@ -40,6 +55,10 @@
       mediaUrl: {
         type: String,
         default: ''
+      },
+      withText: {
```

Comment:
Is this necessary, or could we always render the text, which is better accessibility, and just use CSS in a parent component to not display it if preferred in that context?

Reply:
The socialShare icons are already used across the site on the footer, the exhibition and maybe a few other places I'm not aware of, all of which don't show the text, currently the share modal is the only component which renders the text so the withText sense makes more sense than rendering the text everywhere and then hiding it using css on all other components